### PR TITLE
Remove `cmake-format` linter (more harm than good)

### PR DESCRIPTION
**Pull request by nbolton**
Blocked by: https://github.com/deskflow/deskflow/pull/7703

We lost patience fighting with `cmake-format` and it's arg/line breaking technique, so we're removing it.

Also, the [`cmake-format`](https://github.com/cheshirekow/cmake_format) project looks a bit dead.

### CODE REVIEW

**Review by sithlord48 — CHANGES_REQUESTED** [review]
Undo the changes not related to removing the action please.
no ' for " changes no space / indent changes. 
those should not be mixed in to this commit.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,22 +6,22 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
     tags:
-      - 'v*'
+      - "v*"
```
[the remark this answers is not in the document]
Comment (nbolton) [reply .github/workflows/ci.yml:11]: Accident. That one slipped past me.

**Comment by nbolton** [comment]
> Undo the changes not related to removing the action please. no ' for " changes no space / indent changes. those should not be mixed in to this commit.

Agreed, that was accidental. First time I didn't check the diff in a while... bad luck that it had a bunch of accidental auto linter. Incidentally I have disabled that.

**Review by sithlord48 — APPROVED** [review]
Looks good to me

**Pull request merged**